modules/task/cherami/cherami.go

```diff
@@ -46,15 +46,18 @@ const (
 )
 
 var (
-	_hyperbahnMu         sync.RWMutex
```

Comment:
unfortunately, the retention needs to be a var so I can pass it in as a reference to cherami. Cherami takes int pointers.